src/main/scala/net/snowflake/spark/snowflake/SnowflakeWriter.scala

```diff
@@ -108,7 +108,13 @@ private[snowflake] class SnowflakeWriter(jdbcWrapper: JDBCWrapper) {
                                    params: MergedParameters): DataFrame =
     params.columnMap match {
       case Some(map) =>
-        val names = map.keys.toSeq
+        // Enclose column name with backtick(`) if dot(.) exists in column name
+        val names = map.keys.toSeq.map(name =>
+          if (name.contains(".")) {
+            s"`$name`"
```

Comment:
make sure there is no "`" in the column name of Snowflake tables.

Reply:
confirmed. There is no "`" in the snowflake table column name.